src/components/BookList.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Book from './Book';
+import InputBook from './InputBook';
+
+function BookList() {
+  return (
+    <div>
+      BookList
+      <Book title="The Great Gatsby" author="F. Scott Fitzgerald" />
```

Comment:
- [ ] [OPTIONAL] Great Job passing the title and author as string, I think it'll be better to make use of an array, then use a map function to iterate over the array of books data.. for example

```js
import React from 'react';
import Book from './Book'; // import the Book component

const Books = () => {
  // create an array of books data (this can be fetched from an API or passed as props)
  const books = [
    { id: 1, title: 'The Lord of the Rings', author: 'J.R.R. Tolkien' },
    { id: 2, title: 'Harry Potter and the Philosopher\'s Stone', author: 'J.K. Rowling' },
    { id: 3, title: 'The Hitchhiker\'s Guide to the Galaxy', author: 'Douglas Adams' }
  ];

  return (
    <>
      <div className="books-container">
        Books
        {/* use a map function to render a Book component for each book in the array */}
        {books.map(book => (
          <Book key={book.id} title={book.title} author={book.author} />
        ))}
      </div>
    </>
  );
};

export default Books;
```